sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -490,9 +490,15 @@ def _op_to_template(op: BaseOp, enable_artifacts=False):
     if processed_op.pod_annotations or processed_op.pod_labels:
         template.setdefault('metadata', {})  # Tekton change, don't wipe out existing metadata
         if processed_op.pod_annotations:
-            template['metadata']['annotations'] = processed_op.pod_annotations
+            template['metadata']['annotations'] = {
+                sanitize_k8s_name(key, True, True, True, 253): value
+                for key, value in processed_op.pod_annotations.items()
+            }
         if processed_op.pod_labels:
-            template['metadata']['labels'] = processed_op.pod_labels
+            template['metadata']['labels'] = {
+                sanitize_k8s_name(key, True, True, True, 253): sanitize_k8s_name(value, True, True)
```

Comment:
nit: same as above.

Reply:
I pushed another commit. Now using keyword arguments where there were more than one `True`/`False` flags in a row